misk-launchdarkly/src/main/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlagsModule.kt

```diff
@@ -61,13 +61,17 @@ class LaunchDarklyModule @JvmOverloads constructor(
   ): LDClientInterface {
     // TODO: This shouldn't exist. We should not be exposing LDClientInterface and the only users of this are
     //   apps who're installing this module but not even using the misk or wisp LaunchDarklyFeatureFlags.
-    val baseUri = URI.create(config.base_uri)
     val ldConfig = LDConfig.Builder()
       // Set wait to 0 to not block here. Block in service initialization instead.
       .startWait(Duration.ofMillis(0))
       .dataSource(Components.streamingDataSource())
       .events(Components.sendEvents())
-      .serviceEndpoints(Components.serviceEndpoints().relayProxy(baseUri))
+
+    if (config.base_uri.isNotBlank()) {
```

Comment:
I wonder if this should be an explicit Boolean rather than magic behavior of passing in a null or empty string? Would it be bad if someone accidentally did this and their service unintentionally wasn't using the relay?

Reply:
Certainly, could introduce a `use_relay_proxy` or similar? I think it's not *too* bad in it's current form,. because it's essentially an override for the base LD config. I think if you're using this directly and didn't provide a value, you wouldn't be surprised by it not using the relay.

Reply:
@adrw have added a new (optional) config to explicitly manage this (and defaulted it to `true` for backward compatibility).